01_the_machine_learning_landscape.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<p>this is a sample comment</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yangtang/handson-ml2/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>